source/_integrations/atag.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: Atag
+description: Instructions on how to setup Atag integration.
+logo: atag.png
+ha_category:
+  - Climate
+  - Water heater
+  - Sensor
+ha_release: 0.105
+ha_iot_class: Local Polling
+---
+
+The `Atag` integration allows Home Assistant to connect to [Atag One](https://atag-one.com) thermostats, reporting and setting its status.
+The integration implements the following platforms.
+
+- Climate
+- Water Heater
+- Sensor
+
+## Configuration
+
+The Atag integration can be enabled directly from the config flow UI.
```

Comment:
For a user the "config flow UI" is not something that exists, those are internal names use in the development.

Please describe, from a user perspective, how to set this up.